docs/guides/publish_fetch_packages.md

```diff
@@ -0,0 +1,177 @@
+Similarly as the {{open_aea}} framework, the {{open_autonomy}} framework also works with the concept of **package**: a collection of files that implement a specific component or functionality.
+Publishing a package is simply the process of storing the package into a registry (or in other words, a repository), either locally or remotely.
+
+Remote registries allow to make the package publicly available to other developers. Moreover, it is required that a package be published in a remote registry before [registering the package in the on-chain protocol](./register_packages_on_chain.md).
```

Comment:
It's not strictly required. If the developer wants to keep the code private they can share it via different means.

Reply:
fixed